lib/blockscore/person.rb

```diff
@@ -10,7 +10,7 @@ class Person < Base
 
     attr_reader :question_sets
 
-    def initialize(options = {})
+    def initialize(*)
```

Comment:
I think a more constrained method signature would be `_`. It would assert you can receive "one of anything", rather than "zero or more of anything".